DemoServer/AdditionalLanguages/python/basics/createDocument/metadata.json

```diff
@@ -0,0 +1,112 @@
+{
+    "Slug": "create-document",
+    "SourceFileName": "createDocument.py",
+    "Title": "Create Document",
+    "DescriptionLines": [
+        "**Create a new Python entity and save it as a document** in the Database.",
+        "The document will be stored in the RavenDB database as a JSON object.",
+        "",
+        "The JSON structure is set according to the Python object model passed to the Session.",
+        "Learn more about data modeling in the Document Modeling chapter - see Related Links below."
+    ],
+
+    "Assets": [
+        {
+            "Title": "Document Modeling",
+            "Type" : "link",
+            "Url"  : "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/3-document-modeling#document-modeling"
+        },
+        {
+            "Title": "The Store Command",
+            "Type" : "link",
+            "Url"  : "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/2-zero-to-ravendb#store"
+        }
+    ],
+
+    "Walkthroughs": [
+        {
+            "Title": "Create a new Python object to be saved",
+            "Slug" : "step-1",
+            "DescriptionLines": [
+                "* Create a Python object to be saved.",
+                "",
+                "* By default, _the Python class name_ determines the collection to which the document will belong to in the database.",
+                "  Collections are used to group documents together so that it is convenient to apply some operation to them.",
+                "",
+                "* _The Python class members_ determine the document JSON structure in the database.<br>",
+                "  The JSON serialization for the document fields follows the conventions set on the Document Store",
```

Comment:
In our Python client, JSON serialization process is customizable, but don't use mappers/conventions. Let's talk directly about this